ieee.csl

```diff
@@ -276,7 +306,7 @@
   </citation>
   <!-- Bibliography -->
   <bibliography entry-spacing="0" second-field-align="flush">
-    <layout suffix=".">
```

Comment:
This needs to be removed and reapplied case by case as URLs should not end in a full-stop.